### PR TITLE
Added partial loading example for file system

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description

In this example, a Python script interacts with the Chess.com REST API to extract game data for a specific user on a 
monthly basis. The script retrieves game data for a specified time range, and when additional data is loaded for a 
different time range, it automatically handles de-duplication by deleting any previously loaded files for overlapping time range.

### CODE REVIEW

**Review by AstrakhantsevaAA — COMMENTED** [review]
@dat-a-man Add please `bucket_url` to config.toml and set it up

File: docs/examples/partial_loading/partial_loading.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+keywords: [incremental loading, REST API, dlt, chess.com, data pipeline, backfill management, filesystem]
+---
+
+This script interacts with the Chess.com REST API to extract game data for a specific user on a monthly basis.
+The script retrieves game data for a specified time range, and when additional data is loaded for a different time range,
+it automatically handles de-duplication by deleting any previously loaded files for overlapping time range.
+
+We'll learn:
+
+- How to configure a [REST API source](../general-usage/rest_api_source) using the `dlt` library.
```
Comment (AstrakhantsevaAA) [inline docs/examples/partial_loading/partial_loading.py:14]: fix link: /docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md

File: docs/examples/partial_loading/partial_loading.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+- How to manage and delete old backfill files for de-duplication.
+- How to use [Filesystem](../general-usage/filesystem_destination) as a destination for storing extracted data.
```
Comment (AstrakhantsevaAA) [inline docs/examples/partial_loading/partial_loading.py:16]: fix link

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]

File: docs/examples/partial_loading/partial_loading.py (hunk 1 above)
Comment (dat-a-man) [reply docs/examples/partial_loading/partial_loading.py:14]: updated

File: docs/examples/partial_loading/partial_loading.py (hunk 2 above)
Comment (dat-a-man) [reply docs/examples/partial_loading/partial_loading.py:16]: updated

**Comment by AstrakhantsevaAA** [comment]
@dat-a-man Add please bucket_url to config.toml and set it up

**Comment by dat-a-man** [comment]
Closed as per discussion with Alena.

**Pull request closed without merging**